### ISSUE TITLE
twitch.tv/riotgames not working

### ISSUE THREAD

**Issue by crorens** [chrippa/livestreamer#1354]
I get an error when i try to watch the stream from riotgames. help pls?

**Comment by skulblakka** [comment]
First of all please try to be a little bit more specific. What error are you getting?

Also: twitch.tv/riotgames is offline right now and is hosting nalcs1. In case you want to watch the hosted channel you have to enter their URL (in this case https://www.twitch.tv/nalcs1)

**Comment by crorens** [comment]
Yeah, sorry. I tried to get the error message with screenshot but it dissapears too fast, is there a folder with error messages?

Anyway if I go twitch.tv/riotgames on my web browser the stream starts normally but its like you said, their current stream is /nalcs1. The problem is i didnt know that since when I go to www.twitch/riotgames the stream works like normal. So when i run livestreamer with /nalcs1 it works.
Thanks for the help

**Comment by i-cant-git** [comment]
Open Command Prompt (`cmd.exe`) and type

```
C:\path\to\your\> livestreamer.exe twitch.tv/riotgames source
```

It stream is offline, in this case RiotGames, livestreamer process will terminate. If you want to see the error, you need to start command prompt and start livestreamer there.
